lib/fun_with_flags/store/cache.ex

```diff
@@ -3,12 +3,12 @@ defmodule FunWithFlags.Store.Cache do
   use GenServer
   alias FunWithFlags.Timestamps
   alias FunWithFlags.Flag
+  alias FunWithFlags.Config
 
   @table_name :fun_with_flags_cache
   @table_options [
     :set, :protected, :named_table, {:read_concurrency, true}
   ]
-  @ttl FunWithFlags.Config.cache_ttl
```

Comment:
This comment is not relevant anymore. The cache TTL is not stored in the ETS table alongside the cached data, since https://github.com/tompave/fun_with_flags/commit/effe7a74c964d61f9f3993caae39ccb80f04c4a8.